src/reformat_libyuv.c

```diff
@@ -60,6 +60,18 @@ unsigned int avifLibYUVVersion(void)
 #pragma clang diagnostic pop
 #endif
 
+// libyuv is a C++ library and defines custom types (struct, enum, etc) in a namespace when conditionally compiled with a C++
```

Comment:
1. Nit: We can be explicit and say the namespace is `libyuv`.

2. Nit: Remove "conditionally". It would be more precise to say "when the libyuv header files are included by C++ code". It is important for this comment to say it is the libyuv header files that are compiled with a C++ compiler, because the libyuv implementation (all .cc files) is always compiled with a C++ compiler.

Reply:
Done.